packages/headless/src/api/commerce/common/product.ts

```diff
@@ -18,81 +20,90 @@ export interface BaseProduct {
    *
    * From the `ec_name` field.
    */
-  ec_name?: string;
+  ec_name: string | null;
```

Comment:
@Spuffynism are we sure these appear as null in the JSON returned?

Reply:
Yup, for all of these properties, if the mapped field is not found in the Search API response, the casting to a `String` or a `Double` will result in the field being `null`.